sdk/communication/communication-phone-numbers/karma.conf.js

```diff
@@ -64,7 +64,8 @@ module.exports = function(config) {
       "COMMUNICATION_ENDPOINT",
       "AZURE_CLIENT_ID",
       "AZURE_CLIENT_SECRET",
-      "AZURE_TENANT_ID"
+      "AZURE_TENANT_ID",
+      "COMMUNICATION_SKIP_INT_PHONENUMBERS_TESTS"
```

Comment:
nit: you should add this var to `recordedClient.ts` as well. here: https://github.com/Azure/azure-sdk-for-js/blob/95467bc594aa4196cb04b5b1bbf58599ef14d97e/sdk/communication/communication-phone-numbers/test/public/utils/recordedClient.ts#L35